namespaces/live.cloud-platform.service.justice.gov.uk/iac-fees-dev/resources/api-gateway-iam.tf

```diff
@@ -15,15 +15,15 @@ resource "aws_iam_access_key" "apigw-user" {
 data "aws_iam_policy_document" "apigw" {
   statement {
     actions = [
-      "apigateway:PUT",
-      "apigateway:GET"
+      "apigateway:*",
     ]
 
     resources = [
-      "${aws_api_gateway_rest_api.upload_files_api.arn}/*",
-      aws_api_gateway_rest_api.upload_files_api.arn,
+      "${aws_api_gateway_rest_api.api_gateway.arn}/*",
+      aws_api_gateway_rest_api.api_gateway.arn,
       "arn:aws:apigateway:eu-west-2::/restapis/*",
```

Comment:
`"arn:aws:apigateway:eu-west-2::/restapis/*"` means this iam user can update the contents of all restAPIs in this account. Is it possible to bring this down to only urs?